.github/workflows/cd_push.yml

```diff
@@ -87,7 +96,7 @@ jobs:
 
       - uses: stefanzweifel/git-auto-commit-action@v4
         with:
-          commit_message: Deploy ${{ github.ref_name }} (${{ github.sha }}) to ${{ steps.argocd.outputs.overlay }}
+          commit_message: "[skip ci] Deploy ${{ github.ref_name }} (${{ github.sha }}) to ${{ steps.argocd.outputs.overlay }}"
```

Comment:
We need this because github does not detect that the commit came from CI as we're using an app token so it will do a trigger loop.

Reply:
`[skip ci]` 

is a build in github thing?